easybuild/easyconfigs/t/Theano/Theano-1.1.2-foss-2020b-PyMC.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'PythonPackage'
+
+name = 'Theano'
+version = '1.1.2'
+versionsuffix = '-PyMC'
+
+homepage = 'https://deeplearning.net/software/theano'
+description = """Theano is a Python library that allows you to define, optimize,
+and evaluate mathematical expressions involving multi-dimensional arrays efficiently."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+source_urls = ['https://pypi.python.org/packages/source/T/Theano%(versionsuffix)s']
+sources = ['Theano-PyMC-%(version)s.tar.gz']
+checksums = ['5da6c2242ea72a991c8446d7fe7d35189ea346ef7d024c890397011114bf10fc']
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('SciPy-bundle', '2020.11'),
+    ('mkl-service', '2.3.0'),
+]
+
+download_dep_fail = True
+use_pip = True
+sanity_pip_check = True
+
+fix_python_shebang_for = ['bin/theano-cache', 'bin/theano-nose']
+
+sanity_check_paths = {
+    'files': ['bin/theano-cache'],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
+}
+
+local_blas_warning = "Using NumPy C-API based implementation for BLAS functions"
+sanity_check_commands = [
+    # make sure that warning about falling back to (slow) NumPy C-API for BLAS functions is *not* printed
+    # mkl-service dependency is required to ensure this doesn't happen
```

Comment:
this comment is actually not correct. if the `mkl-service` dependency is not present, the warning is not shown because it will happily use the BLAS that numpy was linked to (OpenBLAS in this case).
see here for more info:
https://theano-pymc.readthedocs.io/en/latest/troubleshooting.html#how-do-i-configure-test-my-blas-library

Reply:
Thanks @smoors for both comments. I had based the easyconfig on the intel one, incorrectly assuming there would be a benefit. I will update accordingly (remove the mkl dependency), it'll make it easier to produce a fosscuda version as well.